### ISSUE TITLE
flot pie draw a label when 100% or 0% in the android do not display a pie chart

### ISSUE THREAD

**Issue by renmm** [flot/flot#1050]
Question:flot pie not dispaly  in the android pad?
Demo:

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>

<script src="http://www.pureexample.com/js/flot/excanvas.min.js"></script>

<script src="http://www.pureexample.com/js/flot/jquery.flot.min.js"></script>

<script src="http://www.pureexample.com/js/flot/jquery.flot.pie.min.js"></script>

<script type="text/javascript">
$(function () { 
    var data = [
        {label: "data1", data:10}
       
    ];
    var options = {
            series: {
                pie: {show: true}
            }
         };

    $.plot($("#example-section16 #670 "), data, options);  
});
</script>

<div id="example-section16">
    <div id="legendPlaceholder"></div>
    <div id="flotcontainer" style="width: 600px;height:200px; text-align: left;"></div>    
</div>
